README.md

```diff
@@ -64,6 +64,12 @@ make install
 make uninstall
 ```
 
+**AUR:**
+
+`yay -S chad_stride`
+
+*if you run into any issues with the AUR package, contact @ginglis13*
```

Comment:
Isn't this unneeded? Doesn't the [PKGBUILD](https://aur.archlinux.org/packages/chad_stride/) on the AUR already list the maintainer?

Reply:
Also, yay is not standard for Arch Linux. You should just list the package name for on the AUR

Reply:
you're right about it being unneeded. If it's that much of a bother, i can remove my name lol

Reply:
also, you are right that there is no standard for an AUR package manager - but yay is the most common.